src/events.test.js

```diff
@@ -349,4 +349,48 @@ describe('event scenarios', () => {
 
     expect(inactiveChild).toEqual('row-a')
   })
+
+  test('node onSelect', () => {
+    const navigation = new Lrud()
+    const onSelectMock = jest.fn()
+
+    navigation.registerNode('root')
+      .registerNode('a', { isFocusable: true, onSelect: onSelectMock })
+
+    navigation.assignFocus('a')
+
+    navigation.handleKeyEvent({ direction: 'ENTER' })
+
+    expect(onSelectMock.mock.calls.length).toEqual(1)
```

Comment:
These `expect`s could be `expect(onSelectMock).toBeCalledTimes(1)` I think?

Reply:
Didn't know about this, but yes! Great shout, done 👍